code/test/CoreTemplateStudio.Core.Test/Helpers/FileHelpersTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using Microsoft.Templates.Core.Helpers;
+using Xunit;
+
+namespace Microsoft.Templates.Core.Test.Helpers
+{
+    [Trait("ExecutionSet", "Minimum")]
+    public class FileHelpersTests
+    {
+        [Theory]
+        [InlineData("")]
+        [InlineData(" ")]
+        [InlineData("   ")]
+        [InlineData(null)]
+        public void GetFileContent_EmptyFilePath_ShouldReturnEmpty(string sourceFile)
+        {
+            var actual = FileHelper.GetFileContent(sourceFile);
+
+            Assert.Empty(actual);
+        }
+
+        [Theory]
+        [InlineData("TestNotExisting.csproj")]
```

Comment:
Is the plan to add more InlineData input for this method?

Reply:
I wanted to test several situations where we can get an error in **File.ReadAllText** but in the end I just added one. Changed this back to [Fact]